hepcrawl/spiders/elsevier_spider.py

```diff
@@ -130,7 +132,8 @@ def _get_package_urls_from_elsevier(self, elsevier_metadata):
         return urls_for_packages
 
     def start_requests(self):
-        request_headers = {'Authorization': 'Basic ' + self.elsevier_authorization_data_base64_encoded}
+        request_headers = {'Authorization': 'Basic ' + self.elsevier_authorization_data_base64_encoded,
+                            'x-els-api-key': self.elsevier_api_key}
```

Comment:
We could have if else depending on which one we set, this way will allow to switch between them :)